tools/build_defs/cmake_script.bzl

```
@@ -34,6 +34,12 @@ def create_cmake_script(
 
     toolchain_dict = _fill_crossfile_from_toolchain(workspace_name, target_os, tools, flags)
     params = None
+
+    keys_with_empty_values_in_user_cache = []
```

nit:

```suggestion
    keys_with_empty_values_in_user_cache = [key for key in user_cache if user_cache.get(key) == ""]
```

and move this down to just before `wipe_empty_values`

---

Thank you for the suggestion, I will apply it. This can NOT be moved down, as the user_cache is changed in the following calls (intentionally).